packages/orchestration/src/orchestration.js

```diff
@@ -51,17 +49,17 @@ const getPower = (powers, name) => {
   return /** @type {OrchestrationPowers[K]} */ (powers.get(name));
 };
 
+export const Proto3Shape = {
+  typeUrl: M.string(),
+  value: M.string(),
```

Comment:
I still think we should have a different key for when value is bytes vs base64 string. e.g. `valueB64`. not a blocker